src/jni_core/src/lib.rs

```diff
@@ -1129,6 +1132,9 @@ pub extern "system" fn Java_com_risingwave_java_binding_Binding_sendSinkWriterEr
     msg: JString<'a>,
 ) -> jboolean {
     execute_and_catch(env, move |env| {
+        if msg.is_null() {
```

Comment:
ditto.

And shall we check all callers to this method to ensure that the msg passed to here must not be null?

Reply:
I have fixed passing msg that could be null in the caller.